admin_app/src/app/user-management/components/UserCard.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import {
+  Avatar,
+  Typography,
+  Button,
+  Card,
+  CardContent,
+  Link,
+  ListItem,
+  IconButton,
+  ListItemIcon,
+} from "@mui/material";
+import LockResetIcon from "@mui/icons-material/LockReset";
+import Edit from "@mui/icons-material/Edit";
+import { Layout } from "@/components/Layout";
+import { sizes } from "@/utils";
+
+interface UserCardProps {
+  username: string;
+  isAdmin: boolean;
+  onResetPassword: () => void;
+  onEditUser: () => void;
+}
+
+const UserCard: React.FC<UserCardProps> = ({
+  username,
+  isAdmin,
+  onResetPassword,
+  onEditUser,
+}) => {
+  const getUserInitials = (name: string) => {
+    const initials = name
+      .split(" ")
+      .map((word) => word[0])
+      .join("")
+      .toUpperCase();
+    return initials;
+  };
+
+  return (
+    <>
+      <Card sx={{ width: 250, padding: 2, textAlign: "center" }}>
+        <CardContent>
+          <Avatar
+            sx={{
+              width: 56,
+              height: 56,
+              margin: "0 auto",
+              bgcolor: "primary.main",
+            }}
+          >
+            {getUserInitials(username)}
+          </Avatar>
+          <Typography variant="h6" mt={2}>
```

Comment:
I think this is causing this error in console

<img width="1057" alt="image" src="https://github.com/user-attachments/assets/96506193-246b-4680-9557-57bb16b6bb6e">

Reply:
Yes fixed it!